### PR TITLE
alt v0.0.1 (new formula)

### PR DESCRIPTION
**Pull request by drewdeponte**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

Added the new formula for the alt command line tool so that it would be easier
to install.

### CODE REVIEW

**Comment by cyphactor** [comment]
When I ran `brew audit --strict --online alt` I got the following.

```
==> brew style alt

1 file inspected, no offenses detected

==> audit problems
alt:
 * Stable: version 0.0.1 is redundant with version scanned from URL
 * Description should use "command-line" instead of "Command line"
 * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
 * GitHub repository too new (<30 days old)
```

File: Formula/alt.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  bottle :unneeded
+
+  depends_on :ruby => "1.9"
+
+  def install
+    bin.install "alt"
+  end
+
+  test do
+    system "#{bin}/alt", "--version"
```
Comment (ilovezfs) [inline Formula/alt.rb]: Please change this to
```
assert_match version.to_s, shell_output("#{bin}/alt --version")
```

**Comment by ilovezfs** [comment]
Are there no docs to install too?

File: Formula/alt.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Alt < Formula
+  desc "command-line utility to find alternate file"
+  homepage "https://github.com/cyphactor/alt"
+  url "https://github.com/cyphactor/alt/archive/v0.0.1.tar.gz"
+  version "0.0.1"
```
Comment (ilovezfs) [inline Formula/alt.rb]: you can remove this line

**Comment by cyphactor** [comment]
@ilovezfs currently the only documentation is https://github.com/cyphactor/alt/blob/master/README.md.

**Comment by ilovezfs** [comment]
@cyphactor you could `doc.install "README.md"` if you want

**Comment by cyphactor** [comment]
So, I think things should be good. When I run audit locally now I get the following:

```
brew audit --strict --online alt
==> brew style alt

1 file inspected, no offenses detected

==> audit problems
alt:
 * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
 * GitHub repository too new (<30 days old)

Error: 2 problems in 1 formula
```

**Comment by tdsmith** [comment]
fyi @ilovezfs, `README.md` (and `LICENSE` and some other files) are automatically installed to `prefix`.

Accepting this formula to homebrew-core seems probably premature; one way you could use Homebrew to install this software is to [maintain your own tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap). The linked document has some advice, but please let us know if you have any questions and we'll be happy to help you set things up.

**Pull request closed without merging**